demo/kserve/scripts/install/kserve-install.sh

```diff
@@ -3,6 +3,7 @@
 # - CHECK_UWM: Set this to "false", if you want to skip the User Workload Configmap check message
```

Comment:
you should start the scripts with this:
```
set -o pipefail
set -o errexit
set -o nounset
set -o errtrace
set -x
```
to make the execution more secure, and more verbose (for troubleshooting), it really helps a lot during the troubleshooting